### PR TITLE
fix: Remove snaps, add highlight to connection list

### PR DESCRIPTION
**Pull request by jonybur**
## **Description**

On-hover behavior for Connected Site on All Permissions screen.
Sites listed on the All Permissions screen should have an [on-hover bg color](https://metamask.github.io/metamask-storybook/?path=/docs/components-ui-listitem--docs) to indicate that they are clickable and will take users to a different screen.

Snaps are removed from Permissions page

[Open in GitHub Codespaces](https://codespaces.new/MetaMask/metamask-extension/pull/24099?quickstart=1)

## **Related issues**

Fixes:

## **Manual testing steps**

1. Go to this page...
2.
3.

### **After**

https://github.com/MetaMask/metamask-extension/assets/11148144/34b4599c-d96c-40be-85a3-c0983df40446

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've completed the PR template to the best of my ability
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-extension/blob/develop/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: ui/components/multichain/pages/permissions-page/index.scss [hunk 1]
```diff
@@ -0,0 +1,2 @@
+@import './connection-list-item';
+@import './permissions-page';
```
Comment (darkwing) [inline ui/components/multichain/pages/permissions-page/index.scss:2]: Needs new line here.

File: ui/components/multichain/pages/permissions-page/permissions-page.js [hunk 2]
```diff
@@ -121,58 +105,11 @@ export const PermissionsPage = () => {
...
-            {Object.keys(sitesConnectionsList).length > 0 && (
-              <>
-                <Text
-                  data-testid="sites-connections"
-                  backgroundColor={BackgroundColor.backgroundDefault}
-                  variant={TextVariant.bodyLgMedium}
-                  textAlign={TextAlign.Center}
-                  padding={4}
-                >
-                  {t('siteConnections')}
```
Comment (salimtb) [inline ui/components/multichain/pages/permissions-page/permissions-page.js:154]: locale should be removed from `en.json` file and all the translations file

File: ui/components/multichain/pages/permissions-page/permissions-page.js [hunk 3]
```diff
@@ -121,58 +105,11 @@ export const PermissionsPage = () => {
...
         />
       ) : null}
-      <Content>
+      <Content padding={0}>
         <Box ref={headerRef}></Box>
-        {shouldShowTabsView ? (
-          <Tabs tabsClassName="permissions-page__tabs">
-            <Tab
-              data-testid="permissions-page-sites-tab"
-              name={t('sites')}
```
Comment (salimtb) [inline ui/components/multichain/pages/permissions-page/permissions-page.js:130]: locale should be removed from en.json file and all the translations file

File: ui/components/multichain/pages/permissions-page/permissions-page.js [hunk 4]
```diff
@@ -121,58 +105,11 @@ export const PermissionsPage = () => {
...
-            {Object.keys(snapsConnectionsList).length > 0 && (
-              <>
-                <Text
-                  data-testid="snaps-connections"
-                  variant={TextVariant.bodyLgMedium}
-                  backgroundColor={BackgroundColor.backgroundDefault}
-                  textAlign={TextAlign.Center}
-                  padding={4}
-                >
-                  {t('snapConnections')}
```
Comment (salimtb) [inline ui/components/multichain/pages/permissions-page/permissions-page.js:168]: same here , should locale should be removed

**Comment by darkwing** [comment]
@jonybur `yarn verify-locales:fix` should fix this PR.

**Review by darkwing — APPROVED** [review]

**Review by NidhiKJha — APPROVED** [review]

**Review by EtherWizard33 — APPROVED** [review]
lgtm

**Pull request merged**